examples/pinproctest/pinproctest.cpp

```diff
@@ -263,37 +231,26 @@ void RunLoop(PRHandle proc)
                         PRDMDDraw(proc,dots);
                     }
                     break;
-                }
                 case kPREventTypeAccelerometerX:
-                {
                     //readData[0] = event->value & 0x3FFF;
                     readData[0] = event->value;
                     break;
-                }
```

Comment:
Braces are typically only used for `case` statements that need to declare local variables, so I removed them as part of the cleanup.